ui/hunter/presets.ts

```diff
@@ -50,14 +31,14 @@ export const DefaultRotation = HunterRotation.create({
 	viperStartManaPercent: 0.1,
 	viperStopManaPercent: 0.3,
 
-	weave: WeaveType.WeaveFull,
+	weave: WeaveType.WeaveNone,
```

Comment:
These changes are fine but I want to clarify a few things:

 - The default settings are also used as the defaults in the raid sim, so its important that they be a good representation of the class so that classes don't seem undervalued compared to others in the raid sim. With regards to weaving and the arrows, you're probably right that these new settings better represent the class.
 - Generally speaking, we don't want to be updating defaults constantly to keep up with the game's progression. That's why phase 1 preset is still the default for most classes. Again this doesn't really apply to the changes here, except maybe the arrows but those are a weird case.

Reply:
I agree whole heartedly, and that's why I thought it was important to make the changes for Hunters at least for posterity sake. 

As for not wanting to keep updating presets for each phase, that makes complete sense. Maybe adding a flag to the options menu to default the presets to whatever phase is selected could be beneficial for future iterations, in WOTLK?